arch/inst/Smrnmi/mnret.yaml

```diff
@@ -18,9 +18,17 @@ access:
   vu: always
 data_independent_timing: false
 operation(): |
-  if (implemented?(ExtensionName::S) && CSR[mstatus].MPP != 2'b11) {
+  if (CSR[mnstatus].MNPP != 2'b11) {
```

Comment:
MPP changed to MNPP. Is that intentional? Why? This needs to be explained in the commit message.